pkg/collector/testdata/collector/containers-builtin-app-cpu-memory-utilization/input_usage.yaml

```diff
@@ -5,33 +5,49 @@ responses:
       containerID: app
       cpuCoresNanoSec: [1000000000, 2000000000, 3000000000, 2000000000]
       memoryBytes: [12000000, 18000000, 22000000, 26000000]
+      avgCPUCoresNanoSec: 1000000000
+      avgMemoryBytes: 2000000000
     - podUID: test-pod-1-deployment-1
       containerID: log-aggregator
       cpuCoresNanoSec: [2000000000, 3000000000, 4000000000, 2000000000]
       memoryBytes: [14000000, 20000000, 22000000, 24000000]
+      avgCPUCoresNanoSec: 2000000000
```

Comment:
The average of the cpuVoresNanoSec values above is  2,750,000,000

Reply:
I don't think that matters for the test.  Having it different makes it clear where the value is pulled from.